packages/ckeditor5-core/src/editor/editorconfig.jsdoc

```diff
@@ -167,9 +165,12 @@
  *
  *	Line break will work only in the extended format when `shouldNotGroupWhenFull` option is set to `true`.
  *
- * * **`toolbar.viewportTopOffset`** &ndash; The offset (in pixels) from the top of the viewport used when positioning a sticky toolbar.
+ * * **`toolbar.viewportTopOffset` (deprecated)** &ndash; The offset (in pixels) from the top of the viewport used when positioning a sticky toolbar.
```

Comment:
This is fine that it's mentioned here. API docs are the place to cover deprecated things.